Patch for implicit declaration of exit in popsquares.c reported by Gentoo QA
patch generated by David J Cozatt

--- savers/popsquares.c	
+++ savers/popsquares.c	
@@ -22,6 +22,7 @@
 #include "config.h"
 
 #include <string.h>
+#include <stdlib.h>
 #include <gdk/gdk.h>
 #include <gdk/gdkx.h>
 #include <gtk/gtk.h>
